bin/wasm-node/javascript/src/index.js

```diff
@@ -27,6 +27,20 @@ export async function start(config) {
   if (Object.prototype.toString.call(config.chain_spec) !== '[object String]')
     throw new SmoldotError('config must include a string chain_spec');
 
+  const logCallback = config.log_callback || ((level, target, message) => {
+    if (level <= 1) {
+      console.error("[" + target + "]", message);
+    } else if (level == 2) {
+      console.warn("[" + target + "]", message);
+    } else if (level == 3) {
+      console.info("[" + target + "]", message);
+    } else if (level == 4) {
+      console.debug("[" + target + "]", message);
+    } else {
+      console.trace("[" + target + "]", message);
+    }
```

Comment:
This is the default implementation if no callback is provided, to get an idea of how it's called.